Pages/Index.cshtml.cs

```diff
@@ -22,13 +22,13 @@ public IndexModel(ILogger<IndexModel> logger)
             _logger = logger;
         }
 
-        public void OnGet()
+        public async Task OnGetAsync()
```

Comment:
Change to asynchronous to prevent some performance issues

Reply:
Good.